x-pack/plugins/security_solution/public/detections/components/rules/rule_info/index.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { UNKNOWN_TEXT } from './translations';
+import { FormattedDate } from '../../../../common/components/formatted_date';
+
+interface CreatedByProps {
+  createdBy?: string;
+  createdAt?: string;
+  ['data-test-subj']?: string;
+}
+
+/**
+ * Created by and created at text that are shown on rule details
```

Comment:
I think I you specify where this component is used, specifying both places would be nicer: on rule details and in the preview panel of an alert flyout